test/examples/ucp_hello_world.c

```diff
@@ -235,6 +258,26 @@ static int run_ucx_client(ucp_worker_h ucp_worker)
     printf("%s", (char *)(msg + 1));
     printf("\n\n---------------------------\n\n");
 
+    if (err_handling_opt.server_failure) {
```

Comment:
Because original code does one way send. I think it's more informative to fail on reply.